provisions/roles/db/tasks/main.yml

```diff
@@ -1,12 +1,13 @@
 ---
 - name: Ensure cccp data dir exists
-  file: path=/var/lib/cccp/db state=directory
+  file: path= "{{ db_backup_host_path }}" state=directory
   sudo: yes
   tags: db
+  when: not test
 
 - name: Set selinux context for datadir for mounting in containers
   sefcontext:
-      target: '/var/lib/cccp/db(/.*)?'
+      target: '{{ db_backup_host_path }}'
```

Comment:
IIUC, in CI environment, we don't create the backup directory. Won't this task fail because we're trying to SELinux context to a dir that doesn't exist?